.github/CODEOWNERS

```
@@ -0,0 +1,20 @@
+# anything with no explicit code owners will be tagged to @core-devs
+*                       @napari/core-devs
+
+# submodules
+napari/_vispy/          @brisvag @melonora
+napari/_qt/             @Czaki @lucyleeow @DragaDoncila
+napari/_app_model/      @lucyleeow
```

```suggestion
napari/_app_model/      @lucyleeow @DragaDoncila 
```

Lucy and I have worked pretty closely on all this, happy to be added here